scripts/docs/generate-transformations.ts

```diff
@@ -151,6 +156,8 @@ ${buildTransformationDocsContent(transformationDocsContent)}
 [dashboard variable]: "/docs/grafana-cloud/ -> docs/grafana/<GRAFANA VERSION>/dashboards/variables"
 
 {{% /docs/reference %}}
+
+[Data frames]: https://grafana.com/developers/plugin-tools/introduction/data-frames/
```

Comment:
@imatwawana I added your changes to the `scripts/docs/generate-transformations.ts` file, and then built the markdown (docs/sources/panels-visualizations/query-transform-data/transform-data/index.md) from there, but it looks like the link in the markdown for `Data frames` doesn't show up after building. Should be on line 922 in the index.md, but isn't. I don't know if this syntax will work.

Reply:
But it does seem to render in the html, so maybe we're good? 

<img width="1440" alt="Screenshot 2023-10-25 at 11 15 54 AM" src="https://github.com/grafana/grafana/assets/46619047/76100193-628a-4075-b90d-629b994e8c46">